test.py

```diff
@@ -107,13 +123,17 @@ def layernorm_fwd_batcher(
 ) -> tuple[Sequence[jax.Array], Sequence[int | None]]:
   x, gamma, beta = batched_args
   x_bdim, gamma_bdim, beta_bdim = batch_dims
-  if not (gamma_bdim is beta_bdim is batching.not_mapped):
-    raise NotImplementedError
+  if not (gamma_bdim is beta_bdim is not_mapped):
+    raise NotImplementedError  # kernel doesn't support gamma or beta batching
   if x_bdim == x.ndim - 1:
     x = jnp.moveaxis(x, -1, -2)
     x_bdim = x.ndim - 2
-  out_bdims = x_bdim, batching.not_mapped, batching.not_mapped
```

Comment:
I think returning the result bdims as `(x_dim, not_mapped, not_mapped)` was a bug in our original PR: the batch dims on `mu` and `rsigma` should match that on `z`!